CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contributing
+
+- [Quick start](#quick-start-)
+- [Build](#build)
+- [Code quality](#code-quality)
+  - [Automatic fixing and formatting](#automatic-fixing-and-formatting)
+  - [Editor integration](#editor-integration)
+- [Testing](#testing)
+  - [Visual regression](#visual-regression)
+- [Continuous deployment](#continuous-deployment)
+- [NPM packages](#npm-packages-)
+  - [Release process](#release-process)
+  - [Local testing](#local-testing)
+- [Icon set](#icon-set)
+
+## Quick start 🚀
+
+```bash
+npm install
+npm start
+```
+
+## Build
+
+- `npm run build` - build H5Web for production
+- `npm run storybook:build` - build the component library's Storybook
+  documentation site
```

Comment:
Nothing about building packages ?

Reply:
Hmm, since you have to go into the `packages` folder and install, I don't think it's the right place. I've added something in the _NPM packages_ section instead.

Reply:
Makes sense :+1: